junifer/data/parcellations.py

```diff
@@ -61,27 +65,28 @@
         "family": "Tian",
         "scale": scale,
         "magneticfield": "7T",
-        "space": "MNI6thgeneration",
+        "space": "MNI152NLin6Asym",
```

Comment:
Same here for the names, it should not contain the space now that we have a dedicated field.

Reply:
To be done in another PR